pkg/operator/ceph/cluster/osd/osd.go

```diff
@@ -892,7 +935,8 @@ func (c *Cluster) deleteOSDDeployment(osdID int) error {
 		}
 	}
 
-	return c.replaceOSD.saveAsConfig(c.context, c.clusterInfo)
+	return nil
+	// return c.replaceOSD.saveAsConfig(c.context, c.clusterInfo)
```

Comment:
no need to save the config here?

Reply:
Removed this comment. Its been saved at a different location now, just after the `deleteOSDDeployment` invocation.